pkg/health/server/prober.go

```diff
@@ -398,6 +421,12 @@ func newProber(s *Server, nodes nodeMap) *prober {
 		}
 		prober.Unlock()
 
+		metrics.Latency.WithLabelValues(
+			node.Name,
+			addr.String(),
+			"icmp",
+		).Observe(rtt.Seconds())
```

Comment:
I think we're still missing the failedProbeLatency case for ICMP?

Reply:
Well, I don't see any error handling in this function. Under what condition should I add failedProbeLatency case there?